css/styles.css

```diff
@@ -385,15 +419,178 @@ hr {
   color: #000;
 }
 
-#bottom-page {
-  margin: 0 6em 1em 6em;
-  padding-bottom: 1rem;
-  border-bottom: 0.2rem solid;
-  color: #000;
+#social-media-footer {
+  margin-bottom: 5rem;
 }
 
-.projects-title {
-  display: flex;
-  justify-content: center;
-  margin-top: 6.5rem;
+@media (min-width: 768px) {
+  /* Head */
+  .hidden-nav {
+    display: inline;
+  }
+
+  #hidden-img {
+    display: none;
+  }
+
+  /* Headline */
+  .headline {
+    flex-direction: inherit;
+    align-items: center;
+    height: 40rem;
+    background-image: none;
+  }
+
+  .headline div {
+    order: 2;
+    width: 49%;
+  }
+
+  .headline h2 {
+    padding: 1rem 0;
+  }
+
+  #headline-social-media {
+    order: 1;
+    width: 9%;
+    display: block;
+  }
+
+  .social-media {
+    list-style-type: none;
+    background: none;
+    gap: 1em;
+    padding: 0;
+  }
+
+  .social-media li {
+    padding: 0.5rem;
+  }
+
+  .social-media li a {
+    height: 2vh;
+  }
+
+  .headline img {
+    order: 3;
+    width: 44%;
+    height: inherit;
+  }
+
+  /* Projects */
+  #works-hr {
+    width: 50%;
+  }
+
+  #projects-title {
+    padding: 10rem 2rem 10rem;
+    font-size: 2.5rem;
+  }
+
+  #flex-projects {
+    display: flex;
+    justify-content: space-around;
+    flex-direction: row;
+  }
+
+  #projects-section-1 {
+    order: 1;
+    width: 100%;
+    display: flex;
+    margin-bottom: -6rem;
+  }
+
+  #projects-section-2 {
+    order: 2;
+    width: 30%;
+  }
+
+  #projects-section-1 div {
+    margin-left: 1rem;
+  }
+
+  #projects {
+    display: flex;
+  }
+
+  #flex-projects h2 {
+    font-size: 1.5rem;
+    padding-top: 1rem;
+  }
+
+  #flex-projects div p {
+    font-size: 1.25rem;
+  }
+
+  .lang-box {
+    padding: 1rem 0;
+    display: flex;
+  }
+
+  .works-lang li {
+    border: 0.05rem solid;
+    padding: 0.5rem 0.5rem;
+    font-size: 1rem;
+  }
+
+  .center-img {
+    margin: -4rem 2rem 0 0;
+  }
+
+  #flex-projects div button {
+    padding: 1rem;
+    font-size: 1rem;
+  }
+
+  /* About me */
+  #about-me ul {
+    flex-direction: column;
+    align-items: normal;
+  }
+
+  /* Languages */
+  #about-me-lang {
+    display: flex;
+    gap: 2rem;
+    justify-content: space-evenly;
+  }
+
+  /* Form */
+  #form {
+    display: flex;
+    gap: 2rem;
+  }
+
+  #form h2 {
+    text-align: left;
+    width: 39%;
+  }
+
+  #contact-form {
+    display: unset;
+    width: 59%;
+  }
```

Comment:
You are using a different font-size and color compared to the design. Also the width, margins, height, etc are slightly different :point_down:  Kindly check and ensure that all are similar to the actual design.


 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-06 06-16-58](https://user-images.githubusercontent.com/65233860/200158530-507cb8da-3599-42f4-915a-5f0b4b2466d0.png)  |  ![Screenshot from 2022-11-06 06-16-45](https://user-images.githubusercontent.com/65233860/200158534-44a7cde0-294c-4cd2-b5e3-1a058accacbc.png)